### PR TITLE
feat(access): access 1.3.0 feature

### PR DESCRIPTION
**Pull request by mawei029**
**What this PR does / why we need it**:
1.support sealed volume operations at access module
2. remove proxy on write process
3. pluggable proxy on read process

**Which issue this PR fixes**:

fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: blobstore/access/controller/cluster.go [hunk 1]
```diff
@@ -107,6 +107,8 @@ type ClusterController interface {
 	All() []*cmapi.ClusterInfo
 	// ChooseOne returns a available cluster to upload
 	ChooseOne() (*cmapi.ClusterInfo, error)
+	// GetClusterClient returns a cluster client
+	GetClusterClient(cid proto.ClusterID) cmapi.APIAccess
```
Comment (sejust) [inline blobstore/access/controller/cluster.go:111]: cid --> clusterID  一个interface 命名规则保持一致

File: blobstore/access/stream.go [hunk 2]
```diff
@@ -450,6 +451,16 @@ func (h *Handler) punishDiskWith(ctx context.Context, clusterID proto.ClusterID,
 	}
 }
 
+func (h *Handler) setVolumeSealed(ctx context.Context, cid proto.ClusterID, vid proto.Vid) {
+	span := trace.SpanFromContextSafe(ctx)
+
+	client := h.clusterController.GetClusterClient(cid)
+	err := client.SetVolumeSealed(ctx, &cmapi.SetVolumeSealedArgs{Vid: vid})
```
Comment (sejust) [inline blobstore/access/stream.go:458]: client  可能是nil， 需要处理。会panic

File: blobstore/access/stream.go [hunk 3]
```diff
@@ -450,6 +451,16 @@ func (h *Handler) punishDiskWith(ctx context.Context, clusterID proto.ClusterID,
 	}
 }
 
+func (h *Handler) setVolumeSealed(ctx context.Context, cid proto.ClusterID, vid proto.Vid) {
+	span := trace.SpanFromContextSafe(ctx)
```
Comment (sejust) [inline blobstore/access/stream.go:455]: 这里记一个warn 日志。

**Review by sejust — CHANGES_REQUESTED** [review]

File: blobstore/access/stream_put.go [hunk 4]
```diff
@@ -441,5 +450,10 @@ func (h *Handler) writeToBlobnodes(ctx context.Context,
 
 	close(writeDone)
 	err = fmt.Errorf("quorum write failed (%d < %d) of %s", writtenNum, putQuorum, blob.String())
+	needMark = true
 	return
 }
+
+func needMarkSealed(code int) bool {
+	return code == errcode.CodeDiskBroken || code == errcode.CodeDiskNotFound
```
Comment (JasonHu520) [inline blobstore/access/stream_put.go:458]: 这里坏盘去掉吧，这里改为释放卷

**Review by JasonHu520 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**